### ISSUE TITLE
SLF4J

### ISSUE THREAD

**Issue by dfa1** [jline/jline3#258]
Hello, 
  thank you for this wonderful library. 

  I'm using jline3 in a project that uses SLF4J as logging facade. It would be nice if I could redirect any log event coming from jline to a dedicated appender.

Anyhow, I can provide a pull request that replaces any org.jline.utils.Log call to SLF4J. Do you see any value in this change? 

There is also SLF4J declared in the dependencyManagement section of the parent pom: are you already planning using SLF4J?

**Comment by gnodet** [comment]
Have you tried using https://www.slf4j.org/legacy.html#jul-to-slf4j ?

**Comment by dfa1** [comment]
yes, jus tried and it is working fine. It outputs even suppressedExceptions.

**Comment by gnodet** [comment]
Yes, the logging is always lazy, so that should not be too much a problem.
